source/interprocedural_analyses/taint/taintConfiguration.ml

```diff
@@ -396,6 +432,11 @@ let parse source_jsons =
     match member "implicit_sources" json with
     | `Null -> { literal_strings = [] }
     | implicit_sources ->
+        check_keys
+          ~required_keys:[]
+          ~valid_keys:["conditional_test"; "literal_strings"]
+          ~current_keys:(Json.Util.keys (Json.Util.member "implicit_sinks" json))
```

Comment:
```suggestion
          ~current_keys:(Json.Util.keys implicit_sources)
```

Reply:
`Json.Util.member` will throw an exception on type error (https://github.com/ocaml-community/yojson/blob/master/lib/util.mli#L61) so this does seem like the better way to go.